src/main/java/uk/gov/hmcts/reform/sscs/tyanotifications/service/LetterUtils.java

```diff
@@ -11,10 +11,7 @@
 
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
-import java.util.Optional;
+import java.util.*;
```

Comment:
does the check style not flag the * imports?

Reply:
Thanks. @nilay913 imports updated to follow repo guidelines.